plugins/discovery-ec2/build.gradle

```diff
@@ -46,13 +46,12 @@ esplugin.bundleSpec.from('config/discovery-ec2') {
 }
 
 tasks.register("writeTestJavaPolicy") {
+  boolean inFips = buildParams.inFipsJvm
+  inputs.property("inFipsJvm", inFips)
+  final File javaPolicy = new File(layout.buildDirectory.asFile.get(), "tmp/java.policy")
+  outputs.file(javaPolicy)
   doLast {
-    final File tmp = file("${buildDir}/tmp")
```

Comment:
the `Project#file` utility method and the `buildParams` extension are not available from task execution phase (e.g. running in doLast) when configuration cache enabled. 

We moved declaring the output file and the inFips boolean into the configuration phase and also declared them as proper inputs/outputs for the `writeTestJavaPolicy` task to mark it properly as up-to-date on unchanged reruns